bin/main

```diff
@@ -1,3 +1,61 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+puts '1 | 2 | 3'
+puts '---------'
+puts '4 | 5 | 6'
+puts '---------'
+puts '7 | 8 | 9'
+
+puts "\nPlayer 1 (X), insert your name:"
+player_1 = gets.chomp # rubocop:todo Naming/VariableNumber
+
+puts "\nPlayer 2 (O), insert your name:"
+player_2 = gets.chomp # rubocop:todo Naming/VariableNumber
+
+puts "\n#{player_1} make your move"
+gets.chomp
+
+game_on = false
+# initial instructions for player
+current_player = 1
```

Comment:
- current_player is supposed to host the instance of the current player(Object), not an number :mag: